Fixed #35894 -- Reduced the usage of "patch" when referring to a code contribution. (#18780)
django/django: docs/internals/contributing/writing-code/working-with-git.txt

Note that the merger is likely to squash the review commit into the previous
commit when committing the code.

> Working on a pull request

Reviewer: I feel like this title is misleading "Testing a pull request locally" maybe (or branch/contribution)

Author: Yes, I agree. Went for "Testing a pull request locally", that seems the most accurate. 👍🏻

=========================

> One of the ways that developers can contribute to Django is by reviewing pull
> requests on GitHub. Those pull requests can be easily integrated into your
> local repository:

Reviewer: ```suggestion
One of the ways that developers can contribute to Django is by reviewing and
testing pull requests. Pull requests can be easily integrated into your local
repository:
```
maybe

Author: Done 👍🏻
